values/roundtriptest/int_test.go

```diff
@@ -41,19 +49,22 @@ func TestPositiveIntDictionaryEncodeAndDecode(t *testing.T) {
 	testEncodeAndDecodeInt(t, data, meta, vals)
 }
 
-func TestNegativeIntDictionaryEncodeAndDecode(t *testing.T) {
+func TestNegativeIntRawSizeEncodeAndDecode(t *testing.T) {
```

Comment:
`TestNegativeVarintEncodeAndDecode`?

Reply:
Changed to `TestNegativeIntVarintEncodeAndDecode`.